scripts/ldmx-env.sh

```diff
@@ -537,6 +539,24 @@ __ldmx_checkout() {
   return 0
 }
 
+###############################################################################
+# __ldmx_compile
+#   Compile ldmx-sw
+###############################################################################
+
+__ldmx_compile() {
+  ldmx . ${LDMX_BASE}/ldmx-sw/scripts/ldmx-compile.sh
+}
+
+###############################################################################
+# __ldmx_recompFire
+#   Recompile ldmx-sw and run fire on the first argument
+###############################################################################
+
+__ldmx_recompFire() {
+  ldmx . ${LDMX_BASE}/ldmx-sw/scripts/ldmx-recompileAndFire.sh $1
```

Comment:
Instead of just the first command line argument, we should pass all of them in case the user is passing arguments to their config on the command line (.e.g `ldmx fire config.py input.root` is one of the more common examples)

```suggestion
  ldmx . ${LDMX_BASE}/ldmx-sw/scripts/ldmx-recompileAndFire.sh $@
```